client/etcd/options.go

```diff
@@ -68,30 +68,29 @@ func (o tlsOptions) SetCACrtPath(ca string) TLSOptions {
 }
 
 func (o tlsOptions) Config() (*tls.Config, error) {
-	var tlscfg *tls.Config
 	if o.cert != "" {
```

Comment:
minor nit: could you go the other route - 

```
if o.cert == "" {
  return nil, nil 
}
// rest of the code
```

Reply:
true, more concise that way